test/test_database.py

```diff
@@ -3,48 +3,44 @@
 Can run all tests in all files by running this from root of TermHub:
     python -m unittest discover
 """
+import os
+import sys
 import unittest
-from enclave_wrangler.objects_api import make_objects_request
-from backend.db.utils import get_db_connection, sql_query_single_col
+from pathlib import Path
+from typing import Set
 
-TEST_SCHEMA = 'test_n3c'
+TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
+PROJECT_ROOT = TEST_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+from enclave_wrangler.objects_api import find_and_add_missing_csets_to_db, get_bidirectional_csets_sets
 
+# TODO: temp
+# Note: Ad hoc import: On 2023/09/12, we found 93 missing csets. The below code was used to import them.
+if __name__ == '__main__':
+    find_and_add_missing_csets_to_db()
 
 class TestDatabaseCurrent(unittest.TestCase):
+    """Tests for database"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Fetch cset IDs to used by further tests"""
+        cls.db_codeset_ids, cls.enclave_codeset_ids = get_bidirectional_csets_sets()
+
+    def test_all_enclave_csets_in_termhub(self):
```

Comment:
## Unit test refactor
I did a little refactor. I split up into 2 different tests. The GitHub action will only run `test_all_enclave_csets_in_termhub()`.

When these 2 tests need to run, there's a `setUpClass()` function that will run first so that it only needs to fetch counts 1 time. This method calls `get_bidirectional_csets_sets()`, which is now one of several new functions in `objects_api.py`.